dongle_app/js/app.js

```diff
@@ -151,4 +151,23 @@ window.onload = function() {
       showQR(dongle);
     }
   );
+
+  function updateOnlineStatus(event) {
+    var ele = document.getElementById('not-on-line');
+    if(navigator.onLine){
+      ele.classList.remove('open');
+    }else{
+      ele.classList.add('open');
+    }
+  }
+
+  window.addEventListener('online',  updateOnlineStatus);
+  window.addEventListener('offline', updateOnlineStatus);
+
+  document.addEventListener("mozvisibilitychange", function() {
+    var wrapperBrowser = document.getElementById('wrapper-browser');
+    var browse = wrapperBrowser.querySelector('#browser');
+    wrapperBrowser.removeChild(browser);
+    wrapperBrowser.classList.add('close');
```

Comment:
Request `document.mozCancelFullScreen`. Actually you need to change this as well in the `fox_player.js`, because currently it's not working as expected [1].

[1] https://github.com/borjasalguero/foxcast/blob/master/dongle_app/libs/fox_player.js#L30